### ISSUE TITLE
AWS provider v4 argument deprecation

### ISSUE THREAD

**Issue by iramello** [umotif-public/terraform-aws-elasticache-redis#20]
When using AWS provider >= v4, we get a warning about deprecated arguments.
In our case:
```
│ Warning: Argument is deprecated
│ 
│   with module.internals.module.redis.aws_elasticache_replication_group.redis,
│   on .terraform/modules/internals.redis/main.tf line 1, in resource "aws_elasticache_replication_group" "redis":
│    1: resource "aws_elasticache_replication_group" "redis" {
│ 
│ Use num_node_groups and replicas_per_node_group instead
│ 
│ (and 23 more similar warnings elsewhere)
```
Will there a new release for address these argument deprecations?
Thanks

Ivan

**Comment by Ohid25** [comment]
I've created this [PR](https://github.com/umotif-public/terraform-aws-elasticache-redis/pull/21) to make the module compatible with `>=4.0.0` - feel free to leave a comment.

**Comment by Ohid25** [comment]
This should now be fixed with [3.0](https://github.com/umotif-public/terraform-aws-elasticache-redis/releases/tag/3.0.0)
